V2.2

xvi10 opened the pull request.

gdev8317 commented on contracts/deposit/DepositUtils.sol:
```diff
@@ -38,6 +38,15 @@ library DepositUtils {
     // @param executionFee the execution fee for keepers
     // @param callbackGasLimit the gas limit for the callbackContract
     struct CreateDepositParams {
+        CreateDepositParamsAdresses addresses;
```
there is typo, should be Addresses instead of Adresses

dmax10 replied:
fixed

xdev10 committed:
fix tests

xdev10 committed:
remove unnecessary dust rounding

xdev10 committed:
use 0 for minAmountLD

xdev10 committed:
allow bridge min amount out to be specified

xdev10 committed:
fix multichain tests

xdev10 committed:
add reentrancy guard to fromController functions

xdev10 committed:
add comments for lzCompose

xdev10 committed:
prevent removal of collateral during decrease position if remaining collateral would be insufficient to cover pending PnL

xdev10 committed:
fix tests

xdev10 committed:
add comments for lzCompose

gdev8317 committed:
ClaimContract: support distribution id

gdev8317 committed:
ClaimContract: support distribution id

gdev8317 committed:
ClaimHandler: add tests

gdev8317 committed:
ClaimHandler: support optional terms signature validation

gdev8317 committed:
ClaimHandler: add events, add removeTerms, optimize tests

gdev8317 committed:
refactor ClaimHandler.depositFunds

gdev8317 committed:
refactor ClaimHandler.withrawFunds

gdev8317 committed:
refactor ClaimHandler.withrawFunds

gdev8317 committed:
refactor ClaimHandler.claimFunds

gdev8317 committed:
ClaimHandler refactor transferClaim

gdev8317 committed:
comments

gdev8317 committed:
remove redundant test
fix event data items length

dmax10 committed:
Re-deploy to arb sepolia

gdev8317 committed:
ClaimHandler: review fixes

gdev8317 committed:
ClaimHandler: add test for bad signature

gdev8317 committed:
add deploy scripts for ClaimHandler, optimize tests speed with loadFixture

gdev8317 committed:
Merge pull request #209 from gmx-io/claim-contract

ClaimHandler contract

gdev8317 committed:
ClaimHandler: add invariant checks to other functions

dmax10 committed:
Verify contracts

gdev8317 committed:
add comment

gdev8317 committed:
ClaimHandler: add tests

dmax10 committed:
Fix params order in testnet script and test multichain flows

TOKEN=ETH npx hardhat run --network sepolia scripts/multichain/bridgeInCrossChain.ts
- Sepolia 0xbb408e20525c1f1b6d62cad41516deadf0c3ab9ef95e98238a6531320c3dd90a
- Arb sepolia 0x3be9b97593ec03f204e119cd76a6e37d6ce02a90c6f3d1f4fb1cb2cee74374e7

TOKEN=GM npx hardhat run --network sepolia scripts/multichain/bridgeInCrossChain.ts
- Sepolia 0x1c90c5276cebb6346db92b19d0309d3094526ff7ccd85a935b75a80b0ccf283a
- Art sepolia 0xb7976f235dce7a5b90e8610562b615cd67c98727b237cfae003ebdd32648fc76

TOKEN=GLV npx hardhat run --network sepolia scripts/multichain/bridgeInCrossChain.ts
- Sepolia 0x9749c72157c27b799abd893024716b974f9de0d5f910821117cc3e8a32dc18b7
- Arb sepolia 0xd621c3cb8ad1ee62b01c220d0a32c3bc6ee5cbd1c2a9f9baa63d66c00223bd04

ACTION_TYPE=Deposit npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia 0x82ddbaa637790820f94a486996f56e30cb2b8faefdbd3b81b0551087ce6583aa
- Arb sepolia
    - Create deposit —> 0xcf3f0f6644cf0da29ecf606e2fcf5c6082003f3115ac7dd2ebfe7d97d586daf2
    - Execute deposit —> 0x32cceb0036a5577e49c80cbe671a78556c730af097085aa20520edd1f2ed8f63

ACTION_TYPE=GlvDeposit npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x7ea967de855ba17f2e803983d9b4cba3da4f91da79b418452c8b36a9a69ad880
- Arb sepolia
    - Create glvDeposit —> 0xaa6bc017fec7d52c845731f0df009954ce70249c233142d8c8da2f815a6f715c
    - Execute glvDeposit —> 0x800e6c0d893a0705598697c7dba25dcf8081aaf4c9156971911f6b1118c93d36

ACTION_TYPE=Withdrawal npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x11d0451a3e8f0bcd8490b0443b16e918d115da9914f3b0c4dc23f0f196f7af37
- Arb sepolia
    - Create withdrawal —> 0x183698409f52c0bc7dae60138cdd4ed562c2af776f2917cf35813a4921bcfce4
    - Execute withdrawal —> 0x9c7db8f06248bb6f62ba3ef26bb5839a4cbd944676c20dabdd4aa0b9bb9af2d2

ACTION_TYPE=GlvWithdrawal npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x2a61c648fd7b9d6789949319229e62a61d349213f89ffaf506d8055c5a19d429
- Arb sepolia
    - Create glvWithdrawal —> 0xeaf3d5873b95605327cdbb9e94661f7711fbb04adde427df9ee0bd5193c51a80
    - Execute glvWithdrawal —> 0xb7faccb3b7afe92832c98a185f6806ef4e0a081b72ed4d1cc2a1888a7933c767

ACTION_TYPE=SetTraderReferralCode npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0xb8974636c33bb10ccd7d143197281b872b98a68149bb9871dc8830dcd62559ab
- Arb sepolia —> 0x18289eee3105149d431bdf6d4ac175876c0cbe04c02cbe4ede5e190903fb0b71

ACTION_TYPE=BridgeOut EXECUTE=Deposit npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x150e70c2959611b6b949f364cc53bcafe3969dc4ca7ba58c2b9b08ad77bcc302
- Arb sepolia —> BridgeOut
    - Create deposit —> 0xdc4cb03198eb031f59c62793610a829b66b2b117ec94685b4bd27b21b2a277fe
    - Execute deposit —> 0x8f1133b45789a77b047e64733b8431fbc9fc4a717b411521b3300634e5e191f4
- Sepolia (GM bridged back to source chain) —> 0x35a7f4b267837986577ecf0d4defcfebc096c244b9ecc35a58cff1abeed2ea8e

ACTION_TYPE=BridgeOut EXECUTE=GlvDeposit npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x8a6224172188f46e855230bbec3d33c4c1d8b9de8892dea0ef9173550f8421de
- Arb sepolia —> BridgeOut
    - Create glvDeposit —> 0xdbebf3af4c8a17f1c67a004672a56a7a91165dbd054f3205c5ea849df3af3390
    - Execute glvDeposit —> 0x7e71913105f828665bc8d8161e4d4f64db735ea26d517f1d122325d1062ce617
- Sepolia (GLV bridged back to source chain) —> 0xfbd47a6d83aa2d86e13789516d56dd3227203e701e81e67df6ea8befb7f54ef4

ACTION_TYPE=BridgeOut EXECUTE=Withdrawal npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0xf615c7a43f3077573405de8bed26c0b55f0ed267140507eecfe898526a82bf12
- Arb sepolia —> BridgeOut
    - Create withdrawal —> 0xd08e6d05dc0a12ce6b8d7cff874aec85c62f497729c6688f58865b22a5d964b2
    - Execute withdrawal —> 0x6e451d993cb59de6985b56191ee8e77558a0a980f3a37aad61725a77ed28e99e
- Sepolia (long / short tokens bridged back to source chain)
    - Eth —> 0x3dd3e02c907f10acfa4cb01bda767854318fc12a601f3add89f3968e234c075d
    - usdc —> 0xb3a2cf5e52b99fbcfd31844b601a0eae648085ad692b5e5bf044719f44d3e398

ACTION_TYPE=BridgeOut EXECUTE=GlvWithdrawal npx hardhat run --network sepolia scripts/multichain/bridgeInComposedMsg.ts
- Sepolia —> 0x83ff9bec8b26f2d8b0242d45090b38d7fc217827d2b4a21d6aeef7d596420193
- Arb sepolia —> BridgeOut
    - Create glvWithdrawal —> 0x14100d81e6775ced70a7d505fc73bea42b19e801917508c0396d2a3929737824
    - Execute glvWithdrawal —> 0x2f7b105ca3c2446bce8ede6822df11d04038436ec19c8c55fceb7aadf9dc39ed
- Sepolia (long / short tokens bridged back to source chain)
    - Eth —> 0x6db26772e235be2e30bf0d5f35ed8f35a5b1b392b38c648df1337b6f0fb160df
    - usdc —> 0x42abe217d8edbdf39179bc27eb5d19cbe9526ec3d434bbe96156d5693f28ef2e

xdev10 committed:
remove unnecessary keys in migration files

xdev10 committed:
update verification script

xdev10 committed:
add updated v2.2 deployments

Commit:
Update values for hedge test

Commit:
Merge remote-tracking branch 'origin/v2.2-branch' into v2.2-branch

xdev10 committed:
update verifications

xdev10 committed:
add spacings to verify logs

dmax10 committed:
Merge branch 'v2.2-branch' into 2025-07-22-deployments-arb-sepolia

dmax10 committed:
Merge branch 'v2.2-branch' into v2.2-additional-tests

dmax10 committed:
Test min remaining collateral on position decrease

xdev10 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

xdev10 committed:
Merge branch 'updates' into v2.2-branch

xdev10 committed:
fix bridge out amounts

Commit:
Update values for hedge test

xdev10 committed:
update deployments

xdev10 committed:
update verifications

xdev10 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

xdev10 committed:
update gelato

xdev10 committed:
update v2.2 deployments

xdev10 committed:
update hardhat

gdev8317 committed:
ClaimContract: fail on transferClaim if fromAccount == toAccount

gdev8317 committed:
ClaimHandler: L-04 Misleading Account Emitted

gdev8317 committed:
ClaimHandler: L-07 Lacking Domain Separator

gdev8317 committed:
ClaimHandler: add distributionId to signature

dmax10 committed:
Merge branch 'v2.2-branch' into 2025-07-22-deployments-arb-sepolia

dmax10 committed:
Merge branch 'v2.2-branch' into v2.2-additional-tests

dmax10 committed:
Merge pull request #211 from gmx-io/2025-07-22-deployments-arb-sepolia

Re-deploy to arb sepolia

gdev8317 committed:
ClaimHandler add globalNonReentrant

xdev10 committed:
update claim handler deployment

xdev10 committed:
add test for invalid signature for minified digest

xdev10 committed:
add changelog

xdev10 committed:
add Guardian audit reports

dmax10 committed:
Merge branch 'updates' into v2.2-branch

dmax10 committed:
Replace setPriceFeed and setDataStream with initOracleConfig

dmax10 committed:
Remove validateTickers from initOracleConfigForTokensUtils

validateTickers is outdated, and is skipping all tokens validations
(uses realtimeFeedId which is not a config field anymore)

dmax10 committed:
Fix initOracleConfigParams default values

dmax10 committed:
Fix description for testnet token

dmax10 committed:
Fix script for outdated tokens

dmax10 committed:
Merge branch 'v2.2-branch' into v2.2-additional-tests

dmax10 committed:
Add optional maxLendableImpactFactor, maxLendableImpactFactorForWithdrawals, maxLendableImpactUsd configs

Values enforced to 0 until V2 old contracts are disabled

dmax10 committed:
Re-deploy to avalancheFuji

dmax10 committed:
Update migrations for avalancheFuji

dmax10 committed:
Add general configs for avalancheFuji

dmax10 committed:
Deploy TimelockConfig

(initially failed with NONCE_EXPIRED)

gdev8317 committed:
add comments

dmax10 committed:
Remove GlvHandler deployment file

GlvHandler contract replaced by GlvDepositHandler, GlvWithdrawalHandler, GlvShiftHandler

dmax10 committed:
Merge branch 'v2.2-branch' into update-initOracleConfigForTokensUtils-script

dmax10 committed:
Add edge oracle as optional param in tokens config

xdev10 committed:
add option to disable claims by distribution id

xdev10 committed:
use config for claim handler test

dmax10 committed:
Verify contracts on avalancheFuji

dmax10 committed:
small refactor

xdev10 committed:
update Reader to return info about the total impact usd for positions

xdev10 committed:
update changelog

dmax10 committed:
Merge branch 'v2.2-branch' into v2.2-additional-tests

dmax10 committed:
Additional multichainTransferRouter validation tests

dmax10 committed:
Whitelist WAVAX on avalancheFuji

dmax10 committed:
Skip "maxLendableImpactFactor", "maxLendableImpactFactorForWithdrawals", "maxLendableImpactUsd" validations for arbitrumSepolia

dmax10 committed:
Update WETH:WETH:USDC market configs on arbSepolia

dmax10 committed:
Update createOrder script to be used on arbitrumSepolia and avalancheFuji

tx arbitrumSepolia --> 0x59f8a6c4b0ce2a16f3c5cca5a9579b285b9983a0403d169770e1125192b76339
tx avalancheFuji --> 0x74438c59b6976187c8291108076f582dd705c65e0282f27c5e56e0e5be4b75da

xdev10 committed:
add erc 1271 support for claims

xdev10 committed:
add signature validation for safe

xdev10 committed:
fix tests

xdev10 committed:
add safe utils

dmax10 committed:
Additional multichainOrderRouter validation tests

xdev10 committed:
update deployments and role configs

xdev10 committed:
update role configs

xdev10 committed:
update contract validation script

dmax10 committed:
Merge pull request #217 from gmx-io/2025-07-30-deployments-avalanche-fuji

Deployments avalanche fuji

dmax10 committed:
Merge pull request #216 from gmx-io/v2.2-max-lendable-impact-configs

Add optional max lendable configs

dmax10 committed:
Merge pull request #215 from gmx-io/update-initOracleConfigForTokensUtils-script

Update initOracleConfigForTokensUtils script

xdev10 committed:
update validate roles script

xdev10 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

xdev10 committed:
Merge branch 'updates' into v2.2-branch

xdev10 committed:
update print tokens script

gdev8317 committed:
add scripts to test ClaimHandler

gdev8317 committed:
add test claim terms

shankars99 commented on contracts/multichain/LayerZeroProvider.sol:
```diff
@@ -0,0 +1,526 @@
+        // msg.sender and `from` need to be validated for `amountLD` and `token` to not be spoofable
+        // additionally, the lzReceive, _credit and _debit functions of the OFT should be checked for any overrides
+        // before the `amountLD` and `token` values can be trusted
+        (address account, uint256 srcChainId, uint256 amountLD, bytes memory data) = _decodeLzComposeMsg(message);
```
in the event that `_decodeLzComposeMsg(...args)` fails (due to a malformed `composeMsg`) the users tokens would be stuck in the composer forever. 

having a `try...catch` on this and a failed guid state is a solution since you know:
1. sender on source 
2. srcEid
3. amount

but this can realistically only happen if the front end and composer are not aligned on `composeMsg` OR if someone builds their own FE and makes a mistake

shankars99 commented on contracts/multichain/LayerZeroProvider.sol:
```diff
@@ -0,0 +1,526 @@
+        // need to manually call the function with the right amount of gas
+        // srcChainId could be zero when returned from _decodeLzComposeMsg, skip follow up actions in this case
+        if (srcChainId != 0 && data.length != 0) {
+            (ActionType actionType, bytes memory actionData) = abi.decode(data, (ActionType, bytes));
```
another instance where `data` can be malformed and therefore not decode which _can_ result in a permanent lock of tokens

xvi10 committed:
Merge pull request #208 from gmx-io/v2.2-additional-tests

V2.2 additional tests (wip)

xdev10 committed:
update role scripts

xdev10 committed:
update scripts

xdev10 committed:
update scripts

xdev10 committed:
update validate deployment utils

xdev10 committed:
add botanix to signer

xdev10 committed:
add printHash script

xdev10 committed:
add scripts

xdev10 committed:
update scripts

xdev10 committed:
update scripts

xdev10 committed:
add scripts

xdev10 committed:
add percentage diffs

gdev8317 committed:
add script to deposit funds into ClaimHandler

xdev10 committed:
add script to set referral storage

gdev8317 committed:
update setClaim, make tokenConfig mandatory

Commit:
fix routescan validation for botanix

Commit:
Merge remote-tracking branch 'origin/v2.2-branch' into v2.2-branch

xdev10 committed:
update scripts

gdev8317 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

gdev8317 committed:
setClaims script: validate claimableAmount is zero

gdev8317 committed:
update printGlpInfo to output txn payload

xdev10 committed:
update role configs

xdev10 committed:
update directories

xdev10 committed:
update folders

xdev10 committed:
add token address to output file

xdev10 committed:
add distributions

gdev8317 committed:
update printClaimHandlerDepositFundsPayload logging

gdev8317 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

gdev8317 committed:
update distributionId type

gdev8317 committed:
generate GLP distribution files, improve file selection in setClaims

gdev8317 committed:
claim script fixes

gdev8317 committed:
improve setClaims script

gdev8317 committed:
auto top up claim admin from safe

gdev8317 committed:
add printClaimTerms

gdev8317 committed:
add GLP distribution files

gdev8317 committed:
update logs

gdev8317 committed:
update logs

gdev8317 committed:
support BATCH_SIZE env var in setClaims script

gdev8317 committed:
optimize setClaims

gdev8317 committed:
support SKIP_SIMULATION in setClaims script

gdev8317 committed:
update logs

gdev8317 committed:
updated setClaims

gdev8317 committed:
deposit BTC GLV for claims

gdev8317 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

gdev8317 committed:
update logs

gdev8317 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

gdev8317 committed:
update comment

gdev8317 committed:
deposit ETH GLV for claims

gdev8317 committed:
add script to send GLV tokens back to safe

gdev8317 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

xdev10 committed:
Merge branch 'updates' into v2.2-branch

xdev10 committed:
update scripts

xdev10 committed:
update scripts

xdev10 committed:
update timelock config

xdev10 committed:
add oracle to oracle provider for token config

dmax10 committed:
Fix pendingImpactAmount

pendingImpactAmount was not stored onto the position and the Reader would return the incorrect value

xvi10 committed:
Merge pull request #220 from gmx-io/v2.2-reader-fix

Fix pendingImpactAmount

xdev10 committed:
update v2.2 deployments

xdev10 committed:
update role configs

xdev10 committed:
update external contracts

xdev10 committed:
update v2.2 deployments

dmax10 committed:
Automatically generate deployment docs on commit, when any contract changes aredetected

- add generateDeploymentDocs.ts to build markdown docs from deployment artifacts
- add generateDeploymentDocs.sh script to detect changed networks and trigger updates for those networks
- add support for arbitrum, avalanche, botanix, arbitrumSepolia, avalancheFuji
- auto update by-network markdown files with contract tables and explorer links
- auto update README with deployment summary and network breakdown
- only update timestamps for networks with actual deployment changes
- set up git hooks to regenerate docs when deployment files change
- add block explorer URL helper to hardhat config

dmax10 committed:
Fix docs update for same-commit changes

- move docs generation from pre-commit to post-commit hook
- auto commit deployment docs updates in a follow-up commit

dmax10 committed:
Fix manual update to correctly display the last updated date for that particular network docs

dmax10 committed:
Refactor output logs

Commit:
collect all rolesToAdd per contractInfo

Commit:
Merge remote-tracking branch 'origin/v2.2-branch' into v2.2-branch

# Conflicts:
#	scripts/updateRoles.ts

xvi10 committed:
Merge pull request #222 from gmx-io/auto-update-deployment-docs

Auto update deployment docs

xdev10 committed:
update changelog

xdev10 committed:
update scripts

xdev10 committed:
add comment

xdev10 committed:
update atomic oracle providers script

xdev10 committed:
add validate functions script

Commit:
add withdraw from position impact pool script

Commit:
add index token oracle price to the withdrawFromPIPool

xdev10 committed:
remove unused structs

xdev10 committed:
Merge branch 'v2.2-branch' of github.com:gmx-io/gmx-synthetics into v2.2-branch

Commit:
set chainlinkPriceFeed as default oracle provider for withdrawalFromPI script

Commit:
Merge remote-tracking branch 'origin/v2.2-branch' into v2.2-branch

dmax10 committed:
Add multichain configs for Arbitrum and Avalanche mainnet

xvi10 committed:
Merge pull request #224 from gmx-io/v2.2-mainnet-multichain-configs

Add multichain configs for Arbitrum and Avalanche mainnet

xdev10 committed:
remove unimplemented code

xdev10 committed:
remove unnecessary comments

xdev10 committed:
move etherscanUtils to utils/explorer

xdev10 committed:
add option to skip read validation in timelock script

dmax10 committed:
Change avalanche block explorer link from snowscan to snowtrace since contracts are not verified on snowscan

xvi10 committed:
Merge pull request #225 from gmx-io/v2.2-deployment-docs-avalanche-explorer-fix

Change avalanche block explorer link from snowscan to snowtrace since…

dmax10 committed:
Add filtering script to distinguish between smart wallets and regular contracts

does not handle proxies yet

dmax10 committed:
Add config validation to check if MultichainOrderRouter received the handler role in ReferralStorage after deployment

dmax10 committed:
Add multichain validation to check multichain providers have been excluded from relay fee payment

dmax10 committed:
Fix FileCache import path

dmax10 committed:
no message

xvi10 committed:
Merge pull request #226 from gmx-io/v2.2-multichain-configs-validations

V2.2 multichain configs validations

xdev10 committed:
update scripts

xdev10 committed:
update scripts

dmax10 committed:
Fix proxies detection, refactoring, add report to md file

dmax10 committed:
Add report

xdev10 committed:
update scripts

xdev10 committed:
update position impact scripts

xdev10 committed:
update scripts

dmax10 committed:
Script update

dmax10 committed:
Merge branch 'v2.2-branch' into glp-distribution-smart-wallets-script

dmax10 committed:
Re-generate report

dmax10 committed:
Re-deploy to arbitrum sepolia

dmax10 committed:
Auto-commit deployment docs for arbitrumSepolia

dmax10 committed:
Update avalanche fuji migrations

dmax10 committed:
Auto-commit deployment docs for avalancheFuji

dmax10 committed:
Re-deploy to avalanche fuji

dmax10 committed:
Update  etherscan verification format (etherscan v2 is using a single api key for all supported networks)

dmax10 committed:
Fix EIP-1271 detection

dmax10 committed:
Flag dolomite addresses

dmax10 committed:
Change output columns

gdev8317 committed:
increase heartbeatDuration on Sepolia

xdev10 committed:
update scripts

dmax10 committed:
Output csv file with isSmartContractWallet and isDolomite filled in, optimize script

dmax10 committed:
Fill the Implementation column

dmax10 committed:
Fill in the implementation address for upgradeable contracts

dmax10 committed:
Save output files to distributions/out to avoid large file commits

dmax10 committed:
fix isDolomite filtering

DOLOMITE_MARGIN is a view function returning an address, with no reverts in it, so no need to check returned data or revert reason

dmax10 committed:
Fix isSmartContractWallet filtering

check the returned value for isValidSignature to be 0x00000000 or 0xffffffff

gmxer committed:
updated arbitrumSepolia priceFeed heartbeats

dmax10 committed:
Update claimableCollateralDelay to 1 day for arbitrum sepolia

dmax10 committed:
Remove checks for implementations

because contracts won't be able to call isValidSignature anyways

dmax10 committed:
Add contracts which need manual checks to report

dmax10 committed:
Remove unused method

dmax10 committed:
Merge branch 'updates' into v2.2-branch

dmax10 committed:
Use same value for minCollateralFactorForLiquidation as it is for minCollateralFactor

dmax10 committed:
Merge pull request #227 from gmx-io/glp-distribution-smart-wallets-script

Glp distribution smart wallets detection script

gdev8317 committed:
update analyzeContractAccounts

gdev8317 committed:
GLP distribution: add distribution files for smart wallets

gdev8317 committed:
remove used distr file

gdev8317 committed:
add comments to setClaims

gdev8317 committed:
deposit glv v1 distribution funds for smart wallets

xvi10 committed:
Merge pull request #233 from gmx-io/pbtc-liquidation-factor

Pbtc liquidation factor for v2.2

gdev8317 committed:
add printTermsSignatureHash script

dmax10 committed:
Add missing LINK configs on avalanche

dmax10 committed:
remove commented conditional

xdev10 committed:
update scripts and data files

xvi10 committed:
Merge pull request #238 from gmx-io/link-configs-avalanche

Link configs avalanche

dmax10 committed:
Merge pull request #231 from gmx-io/2025-08-21-deployments-testnets

2025 08 21 deployments testnets (arb sepolia + avax fuji)

xdev10 committed:
update claimable collateral delay for hardhat
